index.js

```diff
@@ -269,10 +269,26 @@ var PgDriver = Base.extend({
         var constraint = [],
             cb;
 
+        if ((spec.unique || spec.primaryKey) && spec.autoIncrement) {
```

Comment:
Well that would be annoying if they actually do use postgres >= 9.2 :) Currently we have no knowledge about the version that is shared somewhere.